Analytics/Request/BlockingRequestHandler.cs

```diff
@@ -225,19 +225,32 @@ public async Task MakeRequest(Batch batch)
                     if (_client.Config.CompressRequest)
                         content.Headers.ContentEncoding.Add("gzip");
 
-                    var response = await _httpClient.PostAsync(uri, content).ConfigureAwait(false);
+                    HttpResponseMessage response = null;
+                    bool retry = false;
+                    try
+                    {
+                        response = await _httpClient.PostAsync(uri, content).ConfigureAwait(false);
+                    }
+                    catch (System.Threading.Tasks.TaskCanceledException)
+                    {
+                        retry = true;
+                    }
+                    catch (System.Exception e)
+                    {
+                        throw e;
```

Comment:
If we're just throwing, do we need to catch?

Reply:
It was used to debug purpose.
Just want to see what kind of exception can be occur in various network errors.
Ok, I'll remove it.